railgun/test/integration/kongingress_test.go

```diff
@@ -0,0 +1,102 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/go-kong/kong"
+	kongv1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
+	"github.com/kong/kubernetes-ingress-controller/railgun/pkg/clientset"
+	k8sgen "github.com/kong/kubernetes-testing-framework/pkg/generators/k8s"
+)
+
+func TestMinimalKongIngress(t *testing.T) {
+	// FIXME: we're temporarily disabling this test due to timing issues that will be sorted out as part
+	//        of a later iteration in our upcoming testing milestone.
+	//        SEE: https://github.com/Kong/kubernetes-ingress-controller/milestone/16
+	t.Skip()
```

Comment:
I don't like disabled tests. They rot even if they don't cease to compile, and they don't give a clear answer whether a specific bit of functionality is tested or not.

Proposed alternative: extract the disabled test from this PR and add it to a separate draft PR.

Reply:
cb3d991787efc46323dfb7c8ff4cb968c89b6339